index.html

```diff
@@ -6,20 +6,34 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>PATRICK MWANGI PORTIFOLIO</title>
     <link rel="stylesheet" href="style.css">
-		<link rel="stylesheet" href="desktop.css">
+	<link rel="stylesheet" href="desktop.css">
+	
 </head>
 <body>
 	
 	<!-- header -->
 	<header>
-		<div class="logo">Patrick Wanjiru</div>
-		<ul>
-			<li><a href="">Portifolio</a></li>
-			<li><a href="">Contact</a></li>
-			<li><a href="">About</a></li>
-
-		</ul>
-	</header>
+        <nav class="navbar">
+            <a href="#" class="head">Patrick</a>
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="#portfolio" class="nav-link">Portfolio</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#about" class="nav-link">About</a>
+                </li>
```

Comment:
- [ ] Kindly make sure when this ` Portfolio` and `About` is clicked redirect to the part of the page according to which was clicked. For example, if `About` is clicked it must direct to the `About Myself` section.